connectors/netty-connector/src/main/java/org/glassfish/jersey/netty/connector/NettyConnector.java

```diff
@@ -282,6 +283,14 @@ public void run() {
         return settableFuture;
     }
 
+    private String buildPathWithQueryParameters(URI requestUri) {
+        if (requestUri.getRawQuery() != null) {
+            return new StringBuilder(requestUri.getRawPath()).append("?").append(requestUri.getRawQuery()).toString();
```

Comment:
Why not to use `return String.format("%s?%s", requestUri.getRawPath(), requestUri.getRawQuery());`?

And do not forget about copyright year please...